zwave_js_server/model/notification.py

```diff
@@ -11,19 +11,48 @@
 
 
 class NotificationDataType(TypedDict, total=False):
-    """Represent a notification event data dict type."""
+    """Represent a generic notification event data dict type."""
 
     source: Literal["node"]  # required
     event: Literal["notification"]  # required
     nodeId: int  # required
-    notificationLabel: str  # required
+    ccId: int  # required
+
+
+class EntryControlNotificationArgsDataType(TypedDict, total=False):
```

Comment:
If  all keys are required we can set total to True.
```suggestion
class EntryControlNotificationArgsDataType(TypedDict, total=True):
```

Reply:
`eventData` is optional https://zwave-js.github.io/node-zwave-js/#/api/node?id=quotnotificationquot. Will update the PR accordingly